metrics/network/README.md

```diff
@@ -0,0 +1,40 @@
+# Kata Containers Iperf3 and Nuttcp network metrics
+
+Currently, Kata Containers have a series of network performance tests. Use
```

Comment:
I'd simplify to something like:

> Kata Containers provides a series of network performance tests. Running these provides a basic reference for measuring network essentials like bandwidth, jitter, packet per second throughput and latency.

Reply:
I agree with the rewrite suggested by @jodh-intel from a grammar rewrite perspective + an additional series comma after "throughput" :)